Sources/App/entrypoint.swift

```diff
@@ -0,0 +1,15 @@
+import Vapor
+
+@main
+enum Entrypoint {
+    static func main() throws {
+        var env = try Environment.detect()
+        try LoggingSystem.bootstrap(from: &env)
+        
+        let app = Application(env)
+        defer { app.shutdown() }
+        
+        try configure(app)
```

Comment:
That's the idea. I didn't want to try to tackle that at the same time since we haven't resolved stuff like full `AsyncCommand` support (or better yet, integration with `swift-argument-parser`) yet.